desktop/core/src/desktop/api2.py

```diff
@@ -30,7 +30,7 @@
 
 if hasattr(TASK_SERVER_V2, 'get') and TASK_SERVER_V2.ENABLED.get():
   from desktop.celery import app as celery_app
-  from desktop.settings import parse_broker_url
+  from desktop.management.commands.rungunicornserver import parse_broker_url
```

Comment:
Please move `parse_broker_url` to some utils rather than importing `rungunicornserver` everywhere.

Reply:
Moved it to filebrowser utils.